lms/djangoapps/branding/api.py

```diff
@@ -170,6 +170,47 @@ def _footer_social_links():
     return links
 
 
+def _build_support_form_url(full_path=False):
+    """
+    Return the support form path
+
+    Returns url of support form, which can have 3 possible values:
+    - '' if the contact form is disabled (by setting SiteConfiguration
+       CONTACT_US_PAGE = True), by the wich is ignored by the
+       parent function
+    - The normal edx support form path if CONTACT_US_PAGE is not set
+    - CONTACT_US_PAGE if the the user has set a custom URL redirect
+      for support forms
```

Comment:
It seems like it could cause problems down the line to have a value that can be either boolean or string. Someone can easily write code to check for `if contact_us:` and think that means "true/false" instead of "true/false/truthy-string". Could we separate it out into two separate config values -- one for whether or not to display a contact page at all, and the other to indicate if a custom link should be used instead of the default? It's a little more verbose, but I think it's less prone to error.

Reply:
That was my first implementation, but I ended up changing it down the line. We decided to use a single variable because this was such a simple feature and using two variables wasn't strictly necessary. 
I can make those changes so we end up with `CONTACT_US_ENABLE` and `CONTACT_US_CUSTOM_LINK`.

Reply:
Yeah, please do the changes. I realize it's a small thing, but I do think it can confuse folks to have the same variable be very different types. Thank you.